src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml

```diff
@@ -1,6 +1,90 @@
 admin_country_states:
+  path: /country-states
+  methods: [ GET ]
+  defaults:
+    _controller: 'PrestaShopBundle:Admin\Improve\International\State:getStates'
```

Comment:
```suggestion
    _controller: PrestaShopBundle\Controller\Admin\Improve\International\StateController::getStatesAction
```

Reply:
Please use this syntax for routing everywhere, the former one is deprecated and will be removed

Reply:
:+1: